sdk/python/kfp/compiler/_container_builder.py

```diff
@@ -18,32 +18,57 @@
 import os
 import uuid
 
-SERVICEACCOUNT_NAMESPACE = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
+SERVICEACCOUNT_NAMESPACE = '/var/run/secrets/kubernetes.io/serviceaccount/namespace'
+GCS_STAGING_BLOB_DEFAULT_PREFIX = 'kfp_container_build_staging'
 
 class ContainerBuilder(object):
   """
   ContainerBuilder helps build a container image
   """
-  def __init__(self, gcs_staging, namespace=None):
+  def __init__(self, gcs_staging=None, create_bucket_if_not_exist=True, namespace=None):
```

Comment:
nit: do you have a use case that user might set it to False?

Reply:
I'll address this in: https://github.com/kubeflow/pipelines/pull/1776

Reply:
This branch is created on top of that one.

Reply:
Maybe not, I'll remove it.